### PR TITLE
Api call guideline

### PR DESCRIPTION
**Pull request by panipp**
Angular: Api call guideline

### CODE REVIEW

File: angular/api-call/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# API Call
+
+## Environment
+
```
Comment (NorakGithub) [inline angular/api-call/README.md:4]: Could you add more explanation with this one na? I don't understand what is this for.

**Review by NorakGithub — CHANGES_REQUESTED** [review]

File: angular/api-call/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+// api.constant.ts
+
+import { environment } from '../../../environments/environment';
+
+const BaseUrl = environment.baseUrl + '/api';
+```
+
+## API Function
+
+- You should define returned type for your Api function and set function name start with Http method
```
Comment (powerdefy) [inline angular/api-call/README.md]: `Api` -> `API`, `Http` -> `HTTP`

File: angular/api-call/README.md (hunk 1 above)
Comment (panipp) [reply angular/api-call/README.md:4]: @NorakGithub Already updated kaa

File: angular/api-call/README.md (hunk 2 above)
Comment (panipp) [reply angular/api-call/README.md]: @powerdefy Already updated kaa

**Review by piyapan039285 — APPROVED** [review]
ok krub

**Review by NorakGithub — APPROVED** [review]

**Pull request merged**